src/components/common/FullViewSnapshotModal.tsx

```diff
@@ -0,0 +1,205 @@
+          Snapshot Description
+        </FormLabel>
+        <TextField
+          fullWidth
+          margin="normal"
+          id="snapshot-description"
+          label="Snapshot Description"
+          value={snapshotDescription}
+          onChange={(e) => setSnapshotDescription(e.target.value)}
+        />
+        <Typography sx={{ color: 'black' }}>
+          Do you want to use the Google Billing Project associated with this dataset or would you
+          like to select a different one?
+        </Typography>
+        <Box sx={{ marginTop: '8px' }}>
+          <FormLabel
+            sx={{ fontWeight: 600, color: 'black' }}
+            htmlFor="billing-profile-select"
+            required
+          >
+            Google Billing Project
+          </FormLabel>
+        </Box>
+        <JadeDropdown
+          sx={{ height: '2.5rem' }}
+          disabled={billingProfiles.length <= 1}
+          options={uniq(
+            billingProfiles
+              .filter((billingProfile) => billingProfile.profileName !== undefined)
+              .map((billingProfile) => billingProfile.profileName) as string[],
+          )}
+          name="billing-profile"
+          onSelectedItem={(event) =>
+            setSelectedBillingProfile(
+              billingProfiles.find(
+                (billingProfile) => billingProfile.profileName === event.target.value,
+              ),
+            )
+          }
+          value={selectedBillingProfile?.profileName || ''}
+        />
+        <Box sx={{ marginTop: '8px' }}>
+          <FormLabel
+            sx={{ fontWeight: 600, color: 'black' }}
+            htmlFor="select-authorization-domain-select"
+          >
+            Authorization Domain
+            <Box component="span" sx={{ fontWeight: 400, color: 'black' }}>
+              {' '}
+              - (optional)
+            </Box>
+          </FormLabel>
+        </Box>
+        <JadeDropdown
+          sx={{ height: '2.5rem' }}
+          disabled={userGroups ? userGroups.length <= 1 : true}
+          options={userGroups ? userGroups.map((group) => group.groupName) : []}
+          name="Select Authorization Domain"
+          onSelectedItem={(event) => setSelectedAuthDomain(event.target.value)}
+          value={selectedAuthDomain || ''}
+          includeNoneOption={true}
+        />
+        <Box>
+          Authorization Domains restrict data access to only specified individuals in a group and
+          are intended to fulfill requirements you may have for data governed by a compliance
+          standard, such as federal controlled-access data or HIPAA protected data. They follow all
+          snapshot copies and cannot be removed. For more details, see{' '}
+          <Link
+            href="https://support.terra.bio/hc/en-us/articles/360026775691-Overview-Managing-access-to-controlled-data-with-Authorization-Domains#h_01J94P49XS1NE5KE4B3NA3A413"
+            target="_blank"
+          >
```

Comment:
Consider using or extending the `AuthDomain` component, it has a few more tests and would simplify the modal tests.

Reply:
Done! I missed that going in!